mappings/net/minecraft/server/command/ServerCommandSource.mapping

```diff
@@ -4,23 +4,47 @@ CLASS ca net/minecraft/server/command/ServerCommandSource
 	FIELD c output Lbz;
 	FIELD d position Lckp;
 	FIELD e world Luj;
+	FIELD f level I
 	FIELD g name Ljava/lang/String;
 	FIELD h textComponent Lja;
 	FIELD i minecraftServer Lnet/minecraft/server/MinecraftServer;
+	FIELD j silent Z
 	FIELD k entity Lagi;
 	FIELD l resultConsumer Lcom/mojang/brigadier/ResultConsumer;
+	FIELD m entityAnchor Lch$a;
+	FIELD n rotation Lcko;
+	METHOD a withSilent ()Lca;
+	METHOD a withLevel (I)Lca;
 	METHOD a withEntity (Lagi;)Lca;
+	METHOD a withLookingAt (Lagi;Lch$a;)Lca;
```

Comment:
withLookingAt sounds a bit weird... Is it a good name?

Reply:
I wasn't sure how important the `with` prefix was here. It pretty much sets the rotation vector to be looking at the provided entity or position.

Reply:
withLookingAt it is then